src/exchange/diagnostic/diagnostic.html

```diff
@@ -5,80 +5,74 @@
 
     <div class="row" data-ng-if="!ctrl.loaders.accounts">
         <div class="col-md-9">
+            <div class="alert alert-warning"
```

Comment:
I personnally would have replaced it using oui-message (in order to provide example of it)